service/ziti-tunnel/service/ipc.go

```diff
@@ -500,6 +500,10 @@ func serveIpc(conn net.Conn) {
 
 			//save the state
 			rts.SaveState()
+		case "NotifyLogLevelUIAndUpdateService":
+			sendLogLevelAndNotify(enc, cmd.Payload["Level"].(string))
```

Comment:
you don't need `NotifyLogLevelUIAndUpdateService` nor `NotifyIdentityUI`. Both of those should happen already.  `setLogLevel` should broadcast the log level event change and `NotifyIdentityUI` should be changed to send an identity event from toggleIdentity. let's discuss if that's not clear